main.py

```diff
@@ -299,9 +311,25 @@ def logout_view(self):
 
 class SendToSubscribersView(BaseView):
     @login.login_required
-    @expose('/')
+    @expose('/', methods=('GET', 'POST'))
     def index(self):
-        return self.render('sendemail.html')
+        if request.method=='GET':
+            return self.render('sendemail.html')
+        else:
+            jsondata = request.get_json(force=True)
+            emailsubject = (jsondata['subject']).encode("utf8")
+            emailContent = (jsondata['message']).encode("utf8")
```

Comment:
Use consistent capitalization; in Python the standard is underscores and not camel-case